.ci/run_tests.sh

```diff
@@ -7,11 +8,11 @@ createuser --createdb kong
 createdb -U kong kong_tests
 
 if [ "$TEST_SUITE" == "lint" ]; then
-    make lint
+    make lint &>> build.log || (cat build.log && exit 1)
 elif [ "$TEST_SUITE" == "unit" ]; then
-    make test
+    make test &>> build.log || (cat build.log && exit 1)
 elif [ "$TEST_SUITE" == "integration" ]; then
-    make test-integration
+    make test-integration &>> build.log || (cat build.log && exit 1)
 elif [ "$TEST_SUITE" == "plugins" ]; then
-    make test-plugins
+    make test-plugins &>> build.log || (cat build.log && exit 1)
```

Comment:
Yes, the scope of this is only to remove the build noise outputs, tests output are very valuable to have (especially regarding pending tests and the work currently being done around them)